install/kubernetes/helm/subcharts/tracing/values.yaml

```diff
@@ -7,23 +7,54 @@ provider: jaeger
 
 jaeger:
   hub: docker.io/jaegertracing
-  tag: 1.8
-  memory:
-    max_traces: 50000
-  contextPath: /jaeger
-  ingress:
-    enabled: false
-    # Used to create an Ingress record.
-    hosts:
-      - jaeger.local
+  tag: 1.8.2
+  # The spec represents the Jaeger operator's custom resource (CR).
+  # CR docs can be found here: https://github.com/jaegertracing/jaeger-operator#creating-a-new-jaeger-instance
+  # Some example CRs: https://github.com/jaegertracing/jaeger-operator/tree/master/deploy/examples
+  spec:
+    strategy: allInOne
+    allInOne:
+      options:
```

Comment:
it would be helpful if there was a configurable value for `spec.allInOne.image` ( https://github.com/jaegertracing/jaeger-operator/blob/1e134beeb2409beed78d632b858a6691ed4f9cbf/README.adoc#creating-a-new-jaeger-instance) for those of us in air-gapped environments.